Code/RDBoost/Wrap.h

```diff
@@ -370,4 +370,53 @@ struct rdkit_pickle_suite : python::pickle_suite {
   static bool getstate_manages_dict() { return true; }
 };
 
+template<typename T>
```

Comment:
I don't think this can ever be called.

Reply:
I'm not sure what you mean, it's used below by the vec_to_python converter.